core/go/internal/keymanager/dbtypes.go

```diff
@@ -49,11 +49,12 @@ func (t DBKeyVerifier) TableName() string {
 }
 
 var KeyEntryFilters filters.FieldSet = filters.FieldMap{
-	"isKey":       filters.BooleanField("isKey"),
-	"hasChildren": filters.BooleanField("hasChildren"),
+	"isKey":       filters.BooleanField("key_mappings.identifier IS NOT NULL"),
+	"hasChildren": filters.BooleanField("k.p IS NOT NULL"),
 	"parent":      filters.StringField("parent"),
 	"index":       filters.Int64Field(`"index"`),
 	"path":        filters.StringField("path"),
+	"name":        filters.StringField("REGEXP_SUBSTR(key_paths.path, '[^\\.]+$')"),
```

Comment:
@gabriel-indik it's looks like the tests are failing because of this

Reply:
You are absolutely right, addressed the issue.